schema/deploy/tables/application.sql

```diff
@@ -11,6 +11,8 @@ create table ggircs_portal.application (
     reporting_year int references ggircs_portal.reporting_year(reporting_year)
 );
 
+grant all on table ggircs_portal.application to ciip_administrator, ciip_analyst, ciip_industry_user, ciip_guest;
```

Comment:
We should probably have a youtrack card to grep for `grant all on table` but this totally makes sense as a way to not break things